docs/ops/detection/PriorBoxClustered_1.md

```diff
@@ -6,105 +6,100 @@
+  * **Type**: `float[]`
   * **Default value**: 1.0
   * **Required**: *no*
 
 * *clip*
 
-  * **Description**: *clip* is a flag that denotes if each value in the output tensor should be clipped within [0,1].
+  * **Description**: *clip* is a flag that denotes if each value in the output tensor should be clipped within `[0,1]`.
   * **Range of values**:
     * false or 0 - clipping is not performed
-    * true or 1  - each value in the output tensor is within [0,1]
-  * **Type**: boolean
+    * true or 1  - each value in the output tensor is within `[0,1]`
+  * **Type**: `boolean`
   * **Default value**: true
   * **Required**: *no*
 
 * *step (step_w, step_h)*
 
   * **Description**: *step (step_w, step_h)* is a distance between box centers. For example, *step* equal 85 means that the distance between neighborhood prior boxes centers is 85. If both *step_h* and *step_w* are 0 then they are updated with value of *step*. If after that they are still 0 then they are calculated as input image width(height) divided with first input width(height). 
   * **Range of values**: floating point positive number
-  * **Type**: float
+  * **Type**: `float`
   * **Default value**: 0.0
   * **Required**: *no*
 
 * *offset*
 
   * **Description**: *offset* is a shift of box respectively to top left corner. For example, *offset* equal 85 means that the shift of neighborhood prior boxes centers is 85.
   * **Range of values**: floating point positive number
-  * **Type**: float
+  * **Type**: `float`
   * **Default value**: None
   * **Required**: *yes*
 
 * *variance*
 
-  * **Description**: *variance* denotes a variance of adjusting bounding boxes.
+  * **Description**: *variance* denotes a variance of adjusting bounding boxes. The attribute could be 0, 1 or 4 elements.
   * **Range of values**: floating point positive numbers
-  * **Type**: float[]
+  * **Type**: `float[]`
   * **Default value**: []
   * **Required**: *no*
 
-* *img_h (img_w)*
-
-  * **Description**: *img_h (img_w)* specifies height (width) of input image. These attributes are taken from the second input `image_size` height(width) unless provided explicitly as the value for this attributes.
-  * **Range of values**: floating point positive number
-  * **Type**: float
-  * **Default value**: 0
-  * **Required**: *no*
-
 **Inputs**:
 
-*   **1**: `output_size` - 1D tensor with two integer elements `[height, width]`. Specifies the spatial size of generated grid with boxes. Required.
+*   **1**: `output_size` - 1D tensor of type *T_INT* with two elements `[height, width]`. Specifies the spatial size of generated grid with boxes. Required.
 
-*   **2**: `image_size` - 1D tensor with two integer elements `[image_height, image_width]` that specifies shape of the image for which boxes are generated. Optional.
+*   **2**: `image_size` - 1D tensor of type *T_INT* with two elements `[image_height, image_width]` that specifies shape of the image for which boxes are generated. Optional.
```

Comment:
Are these inputs of the same integer type? Because the previous version doesn't restrict this

Reply:
Yes, they are, for example this part could give us a hint about this:
```    
    template <element::Type_t ET>
    bool evaluate(const HostTensorPtr& arg0,
                  const HostTensorPtr& arg1,
                  ...
    {
        runtime::reference::prior_box_clustered(arg0->get_data_ptr<ET>(),
                                                arg1->get_data_ptr<ET>(),
                                                ...
    }